### PR TITLE
fix: in error message must show the values

### PR DESCRIPTION
**Pull request by Xelzs**
Hi ! Thank you for this complete library.

I have fixed the `in` error message by adding the `values` placeholder, like you said in your [docs](https://www.simple-body-validator.com/error-messages/customizing-error-messages#specifying-custom-message-for-a-rule).

### CODE REVIEW

**Comment by Stallos11** [comment]
Much needed

**Comment by jadKhoury1** [comment]
Hello, thank you for opening the pull request, you are right I missed the **values** placeholder. However, the correct message should be **The :attribute must be one of the following :values** without the **types**. In the doc I added the **types** word just for the example, to show that you can create custom messages.

And based on the guidelines that we specified (https://github.com/jadKhoury1/simple-body-validator/blob/main/CONTRIBUTING.md), all the unit tests must pass before approving the pull request. 

So just make sure please that the tests are passing and to add the correct message. Thank you.

File: src/locales/en.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ export default {
         array: 'The :attribute must have :value items or more.',
         object: 'The :attribute must have :value items or more.',
     },
-    in: 'The selected :attribute is invalid.',
+    in: 'The :attribute must be one of the following types: :values.',
```
Comment (jadKhoury1) [inline src/locales/en.js:42]: No need for the **types:** in the error

**Comment by Xelzs** [comment]
Done ✅ 
If you want, I can also add in another PR, some CI/CD to automate your release/test workflow.

**Review by jadKhoury1 — APPROVED** [review]

**Comment by jadKhoury1** [comment]
Thank you for the update I will publish your change in the next release 1.3.3 

As for the other PR, I think it is a great idea to automate the workflow.

**Pull request merged**